packages/@aws-cdk/cfnspec/spec-source/713_ElastiCache_User_patch.json

```diff
@@ -0,0 +1,19 @@
+{
+  "ResourceTypes": {
+    "AWS::ElastiCache::User": {
+      "patch": {
+        "description": "Remove the Authentication and UserGroupIds attributes from the AWS::ElastiCache::User resource type, as they have a complex type",
```

Comment:
I vaguely remember why this is an issue, but you remind refreshing my memory? why are complex types a problem?

Reply:
Because our current code generation in `cfn2ts` doesn't handle them.

Reply:
Ha, ok.